Soma-Technologies-Inc/visitAfrica-backend
# #175494090 notification for approved request

#### What does this PR do?

- Notification for approved request
#### Description of Task to be completed?

- This PR notify the requester when his/her request has been approved.
- It also get all the notifications approved.
#### How should this be manually tested?
- Clone the repo using `git clone https://github.com/Soma-Technologies-Inc/visitAfrica-backend.git`
- Run `npm install`
- Run `git pull `to make sure you have updated code
- Run `npm run migrate` to create migrations for the database
- Run git checkout `ft-notification-approved-request-175494090` to navigate to my branch
- Run `npm run startDev`
- Open postman 
- navigate to login route POST `http:localhost:3000/api/auth/login` and login to get token that will be used to approve the request of the user.
- use these body data { "email": "Benie@gmail.com", "password": "GoodG@12" }
- Then in the header in Authorization put the token got from the logging in using bearer then put the token.
- Then if you want to approve the status while creating the notification use PATCH `http://localhost:3000/api/trip/editstatus/:id`
- Then type `http://localhost:3000/api/notification/approved` using GET to get all the notifications approved.
#### Any background context you want to provide?
N/A
#### Screenshots (if appropriate)
The manager approving the status
![Screenshot from 2021-01-07 21-23-11](https://user-images.githubusercontent.com/73790729/103935015-a0059d00-512e-11eb-9e0d-c9cd0024e96d.png)
getting the approved notifications
![Screenshot from 2021-01-07 21-24-06](https://user-images.githubusercontent.com/73790729/103935153-d2af9580-512e-11eb-981d-fbd24c427999.png)

#### Questions:
N/A
